code/script.js

```diff
@@ -31,17 +34,140 @@ const showMessage = (message, sender) => {
 }
 
 // Starts here
+
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
+  showMessage(`Hey! Willkomen bei Nadelöhr! Mein Name ist Kira 👋 Wie heisst du?`, 'bot')
   // Just to check it out, change 'bot' to 'user' here 👆
 }
 
-// Set up your eventlisteners here
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
-setTimeout(greeting, 1000)
+setTimeout(greeting, 1000);
+
+const handleNameInput = (event) => {
+    event.preventDefault()
+
+    const name = nameInput.value
+    showMessage(nameInput.value, 'user')
+    nameInput.value=''
+
+    setTimeout(() => showWelcomeMessage(name), 1200)
+}
+
+const showWelcomeMessage = (userName) => {
+    
+  showMessage(`Toll das du hier bist ${userName}!!`, 'bot')
+
+  setTimeout(() => showKursOptions(name), 2000)
+
+}
+
+const showKursOptions = (userName) => {
+
+  showMessage(`Wie sind deine heutigen Nähskills ${userName}?`, 'bot')
+
+  inputWrapper.innerHTML = `
+  <button id="nähenBeginnerBtn">habe noch nie zuvor genäht</button>
+  <button id="nähenFortgeschrittenBtn">ganz okay</button>
+  <button id="nähenExpertenBtn">ziemlich gut!</button>
```

Comment:
Same with id's :)

Reply:
Although, I can guess what "ganz" means because in Swedish we have the word "ganska" for "pretty/quite" ;)